cmd/helm/install.go

```diff
@@ -264,23 +264,33 @@ func (i *installCmd) run() error {
 		helm.InstallDisableHooks(i.disableHooks),
 		helm.InstallTimeout(i.timeout),
 		helm.InstallWait(i.wait))
-	if err != nil {
-		return prettyError(err)
+
+	// If there is an error while waiting, make a call without waiting to get the release content
+	if (resp == nil || resp.Release == nil) && i.wait {
```

Comment:
Should this be wrapped with `settings.Debug`?

Reply:
Actually, I looked some more into this. I think it should be `i.printRelease` which is where it prints debug